sr_demos/scripts/sr_demos/demo.py

```diff
@@ -154,345 +172,376 @@ def run(self):
+        self.execute_command_check('l_int_all', 0.5)
+        self.execute_command_check('l_ext_all', 0.5)
+        self.execute_command_check('l_int_ff', 0.5)
+        self.execute_command_check('l_int_mf', 0.5)
+        self.execute_command_check('l_int_rf', 0.5)
+        self.execute_command_check('l_int_lf', 0.5)
+        self.execute_command_check('l_zero_all', 0.5)
+        self.execute_command_check('l_spock', 0.5)
+        self.execute_command_check('l_zero_all', 0.5)
+        self.execute_command_check('pre_ff_ok', 1.0)
+        self.execute_command_check('ff_ok', 0.9)
+        self.execute_command_check('ff2mf_ok', 0.4)
+        self.execute_command_check('mf_ok', 0.9)
+        self.execute_command_check('mf2rf_ok', 0.4)
+        self.execute_command_check('rf_ok', 0.9)
+        self.execute_command_check('rf2lf_ok', 0.4)
+        self.execute_command_check('lf_ok', 0.9)
+        self.commander.move_to_named_target("open")
+
+        for _ in range(0, 3):
+            for movement in flex_ext_sequence:
+                self.execute_command_check(movement, 0.2)
+
+        self.execute_command_check('pre_ff_ok', 1.0)
+        self.execute_command_check('ff_ok', 3.3)
+        self.execute_command_check('ne_wr', 1.1)
+        self.execute_command_check('nw_wr', 1.1)
+        self.execute_command_check('sw_wr', 1.1)
+        self.execute_command_check('se_wr', 1.1)
+        self.execute_command_check('ne_wr', 0.7)
+        self.execute_command_check('nw_wr', 0.7)
+        self.execute_command_check('sw_wr', 0.7)
+        self.execute_command_check('se_wr', 0.7)
+        self.execute_command_check('zero_wr', 0.4)
+        self.commander.move_to_named_target("open")
+        rospy.loginfo("Standard demo completed")
+
+    def rock_paper_scissors(self):
+        '''
+            Runs the Rock, Paper, Scissors demo
+        '''
+        rospy.loginfo("Rock, Paper, Scissors demo started")
+        self.commander.move_to_named_target("open")
+
+        rospy.loginfo("Welcome to the Rock, Paper, Scissors game!")
+        rospy.loginfo("The hand will count down from 3 and then you will make a gesture.")
+        rospy.loginfo("Rock is a fist, Paper is a flat hand, and Scissors is a peace sign.")
+        rospy.loginfo("Ready?")
+        rospy.sleep(3)
+
+        # Count down
+        self.execute_command_check('count_down_3', 1.0)
+        rospy.loginfo("3")
+        self.execute_command_check('count_down_2', 1.0)
+        rospy.loginfo("2")
+        self.execute_command_check('count_down_1', 1.0)
+        rospy.loginfo("1")
+        self.execute_command_check('count_down_0_ff', 1.0)
+        self.execute_command_check('count_down_0_th', 1.0)
+        rospy.loginfo("Make your gesture!")
+
+        # Select a pose at random
+        poses = ['rock', 'paper', 'scissors']
+        pose = random.choice(poses)
+        self.execute_command_check(pose, 1.0)
+        rospy.loginfo("The hand made the {} gesture!".format(pose))
```

Comment:
```suggestion
        rospy.loginfo(f"The hand made the {pose} gesture!")
```

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b